### DISCUSSION TITLE
Hausdorf Distance not in summary.json

### DISCUSSION THREAD

**Discussion by i-moes** [MIC-DKFZ/nnUNet#1316] [Q&A] [answered]
Hi, I want to compute the HD for my segmentation task. I noticed that this is computed in metrics.py. However, the HD, AHD, and HD95 are not returned in the summary.json files. Do I need change something in order to have this metric printed in there as well? 
Thank you!

**Comment by FabianIsensee** [comment, marked as answer]
Use this function:
https://github.com/MIC-DKFZ/nnUNet/blob/7f1e273fa1021dd2ff00df2ada781ee3133096ef/nnunet/evaluation/evaluator.py#L446

with `advanced=True`

**Reply by i-moes** [reply]
Thank you Fabian. I did use nnUnet_evaluate_folder. But, without advanced=True.   I do, however, run it on a GPU via sbatch/.sh, where -h does not state an input option for the advanced setting. Do you know whether one can still run it with this advanced setting in this way?

**Reply by FabianIsensee** [reply]
Write your own eval script with using the function I linked above

**Reply by i-moes** [reply]
That makes total sense! Got it, thank you Fabian.
